ros_packages/voice_assistant/voice_assistant/chat.py

```diff
@@ -81,51 +83,104 @@ def chat(self, goal_handle: ServerGoalHandle):
             request: Chat.Goal = goal_handle.request
             chat_id: str = request.chat_id
             content: str = request.text
+            generate_code: bool = request.generate_code
 
             # get the personality that is associated with the request chat_id from the pib-api
             with self.voice_assistant_client_lock: 
                 successful, personality = voice_assistant_client.get_personality_from_chat(chat_id)
             if not successful:
                 self.get_logger().info(f"no personality found for id {chat_id}")
                 goal_handle.abort()
-                return Chat.Result()
+                return Chat.Result() # TODO
```

Comment:
Is the ToDo part of this story? If not, add a description for the future

Reply:
removed